sphinx/screens/dashboard/dashboard/src/main/res/values-es/strings.xml

```diff
@@ -84,4 +84,7 @@
     <string name="dashboard_connect_retrieve_person_data_error">No se ha podido obtener la información de la persona</string>
     <string name="dashboard_connect_message_empty">El mensaje no puede ser vacío</string>
     <string name="dashboard_connect_generic_error">Ocurrió un error al intentar conectar con el usuario. Por favor volvé a intentarlo más tarde</string>
+
+    <string name="dashboard_network_disconnected_node_toast">Disconnected node</string>
+    <string name="dashboard_network_connected_node_toast">Connected node</string>
```

Comment:
Please revert

The Spanish translations were ok, so those should stay in Spanish (I'm doing them normally and the ones you used initially were right). 
- Nodo Conectado
- Nodo Desconectado